Emma/emma_libs/configuration.py

```diff
@@ -124,7 +144,23 @@ def __readAddressSpacesJson(path):
         """
         # Load the addressSpaces file
         addressSpaces = Emma.shared_libs.emma_helper.readJson(path)
-
+        for memType in addressSpaces["memory"].keys():
+            if START not in addressSpaces["memory"][memType].keys():
+                sc().error("The " + path + " does not have the key: " + START)
+            else:
+                try:
+                    int(addressSpaces["memory"][memType][START], 16)
+                except ValueError:
+                    sc().error("the start address " + addressSpaces["memory"][memType][START] + " in " + path + " is not valid")
```

Comment:
Typo:
```suggestion
                    sc().error("The start address " + addressSpaces["memory"][memType][START] + " in " + path + " is not valid")
```

Also use f-strings for "prints"